pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -76,10 +76,10 @@ func (o *capability) Close() error {
 }
 
 type workflowConfig struct {
-	AggregationMethod       string `mapstructure:"aggregation_method"`
-	AggregationMethodConfig map[string]any
-	Encoder                 string
-	EncoderConfig           map[string]any
+	AggregationMethod string         `mapstructure:"aggregation_method"`
+	AggregationConfig map[string]any `mapstructure:"aggregation_config"`
+	Encoder           string         `mapstructure:"encoder"`
+	EncoderConfig     map[string]any `mapstructure:"encoder_config"`
```

Comment:
To just `Aggregation` and `Encoder` (`Config` seems redundant), which takes in a map with config.

Reply:
I like it but do you mind if I change in a separate PR? That change will span multiple files.